pkg/noobaaaccount/reconciler.go

```diff
@@ -373,6 +375,30 @@ func (r *Reconciler) CreateNooBaaAccount() error {
 		r.Logger.Errorf("got error on NooBaaAccount creation. error: %v", err)
 		return err
 	}
+	// create join secret conatining auth token for remote noobaa account
+	annotationValue, exists := util.GetAnnotationValue(r.NooBaa.GetAnnotations(), "remote-operator")
```

Comment:
I see now that the annotation is on the noobaa CR and not the noobaaAccount CR. In that case, what's the difference between `remote-operator` and `MulticloudObjectGatewayProviderMode`?

Reply:
The `remote-operator` is an annotation which is being added to the account CR by the ocs-operator while we are on the provider side in order to indicate when we onboard a new consumer and therefore need a new auth token to be added to the account. The `MulticloudObjectGatewayProviderMode` annotation is being added to the Noobaa CR while we are running remotely and need to indicate that we only wish to deploy the operator and connect to a remote cluster.

Reply:
But here you check the annotaion on the noobaa CR and not the account CR (`r.NooBaa.GetAnnotations()`)

Reply:
you are correct it's a mistake, will fix.